### ISSUE TITLE
The log of ASR training  is different from LM training's when arg.resume was setted

### ISSUE THREAD

**Issue by fanlu** [espnet/espnet#1231]
the validation loss and acc caculated in ASR training will be recorded in exp/**/results/log generated by chiner.
How should I retain these values trained previously when I set resume params?
All training loss and validation ppl values in LM training will be retaioned when args.resume was setted. the log is below.
```
# lm_train.py --config conf/lm.yaml --ngpu 4 --backend pytorch --verbose 1 --outdir exp/train_rnnlm_pytorch_lm --tensorboard-dir tensorboard/train_rnnlm_pytorch_lm --train-label data/local/lm_train/train.txt --valid-label data/local/lm_train/valid.txt --resume exp
/train_rnnlm_pytorch_lm/snapshot.ep.20 --dict data/lang_1char/train_sp_units.txt --dump-hdf5-path data/local/lm_train 
free gpu: 0,1,2,3
2019-09-30 10:51:25,607 (lm_train:140) INFO: ngpu: 4
2019-09-30 10:51:25,615 (lm_train:143) INFO: python path = (None)
2019-09-30 10:51:25,770 (lm_train:160) INFO: backend = pytorch
2019-09-30 10:51:25,957 (lm:193) INFO: torch version = 1.0.1.post2
2019-09-30 10:51:25,996 (deterministic_utils:24) INFO: torch type check is disabled
2019-09-30 10:51:28,066 (lm_utils:39) INFO: loading binary dataset: data/local/lm_train/valid.txt.h5
2019-09-30 10:51:28,468 (lm_utils:39) INFO: loading binary dataset: data/local/lm_train/train.txt.h5
2019-09-30 10:51:41,339 (lm:207) INFO: #vocab = 4573
2019-09-30 10:51:41,343 (lm:208) INFO: #sentences in the training data = 5130732
2019-09-30 10:51:41,343 (lm:209) INFO: #tokens in the training data = 90376757
2019-09-30 10:51:41,343 (lm:210) INFO: oov rate in the training data = 0.00 %
2019-09-30 10:51:41,343 (lm:211) INFO: #sentences in the validation data = 1000
2019-09-30 10:51:41,343 (lm:212) INFO: #tokens in the validation data = 19716
2019-09-30 10:51:41,343 (lm:213) INFO: oov rate in the validation data = 0.04 %
2019-09-30 10:51:41,343 (lm:219) INFO: batch size is automatically increased (64 -> 256)
2019-09-30 10:51:45,099 (lm:224) INFO: #iterations per epoch = 20064
2019-09-30 10:51:45,099 (lm:225) INFO: #total iterations = 802560
2019-09-30 10:51:49,000 (lm:241) INFO: writing a model config file to exp/train_rnnlm_pytorch_lm/model.json
2019-09-30 10:51:49,208 (lm:289) INFO: resumed from exp/train_rnnlm_pytorch_lm/snapshot.ep.20
epoch       iteration   main/loss   perplexity  val_perplexity  elapsed_time
0           100         118.749     1018.57                     24.1656       
0           200         108.441     259.358                     39.4158       
0           300         83.7734     128.565                     53.0459       
0           400         93.7078     77.0425                     67.4006       
0           500         70.4141     53.2415                     80.813        
0           600         84.1809     47.4972                     95.8007
```

**Comment by sw005320** [comment]
Oh, I did not notice this issue. Thanks for pointing out it. I actually thought that we cannot retain the information but LM training can do it... Hmm. I'll take a look at the difference of the resume function between ASR and LM.
